Engine/source/platformPOSIX/POSIXCPUInfo.cpp

```diff
@@ -20,7 +20,8 @@
 // IN THE SOFTWARE.
 //-----------------------------------------------------------------------------
 
-#ifndef __APPLE__
+//XXTH orig #ifndef __APPLE__
+#if not defined (__APPLE__) //and not defined (__FreeBSD__)
```

Comment:
`#if not defined (__APPLE__)` is not a valid/standard preprocessor expression.

I believe you meant
`#if !defined (__APPLE__)`